src/bridge/graphs/peg_in.rs

```diff
@@ -115,6 +122,8 @@ pub struct PegInGraph {
     pub depositor_public_key: PublicKey,
     depositor_taproot_public_key: XOnlyPublicKey,
     depositor_evm_address: String,
+
+    connectors: PegInConnectors,
```

Comment:
This is adding a level of indirection whenever accessing a connector. I suggest we keep the connectors directly in the graph struct, like we do transactions.

Reply:
sure, done